### DISCUSSION TITLE
Support ext for extension short hand

### DISCUSSION THREAD

**Discussion by GordonBeeming** [community/community#9958] [Other Feature Feedback, Questions, & Ideas] [answered]
It would be nice for the extension filter to support ext to be shorter but then also match what's coming across from Azure DevOps
Currently we are told that ext is unrecognized 
![image](https://user-images.githubusercontent.com/5680199/148839456-4fccee4a-a102-48ca-9e30-be9e9e66bc64.png)

**Comment by look** [comment, marked as answer]
Thanks for the feedback! We've decided to go in a different direction for now by expanding the `path:` qualifier to handle all path-related searches and removing the `extension:` qualifier. Details can be found on the [Syntax page](https://cs.github.com/about/syntax#path), but tldr, you can now do the example search as `Highlight path:*.cs".

This isn't what you suggested, but it is shorter, so hopefully that's helpful! The reason we are making this change is to keep the query language compact so it causes fewer conflicts with search terms.

As always, try it out and let us know what you think.

**Reply by GordonBeeming** [reply]
I like the simplification, path roles off the fingers easily, I'll get used to it 😅
Thank you for the update @look ☺️
